common/src/main/java/xyz/wagyourtail/jsmacros/client/api/classes/Draw2D.java

```diff
@@ -59,19 +69,19 @@ public int getWidth() {
      */
     @Override
     public int getHeight() {
-        return mc.getWindow().getScaledHeight();
+        return heightSupplier.getAsInt();
     }
 
     /**
      * @since 1.0.5
      * @see IDraw2D#getTexts()
      */
     @Override
-    public List<RenderCommon.Text> getTexts() {
-        List<RenderCommon.Text> list = new LinkedList<>();
```

Comment:
That happened because I first moved all RederCommon elements into their own class, the file has nearly 5000 lines of code, and then put them back because of your suggestion to move them in another PR.
So it's hopefully only a temporary thing and will soon be resolved.
That's also why I won't undo this for now if you don't insist on it.